src/features/Courses/CoursesDetailPage/index.scss

```diff
@@ -0,0 +1,48 @@
+@import 'assets/variables.scss';
+@import 'assets/colors.scss';
+
+.card-container {
+  height: 100px;
+  padding: 0 1.5em;
+  padding-left: 2em;
+  padding-right: 4em;
+  background-color: $color-white;
+  border-radius: $border-radius-1;
+  box-shadow: 0px 3px 12px 0px rgba(0, 0, 0, 0.16);
+
+  .title {
+    font-size: 15px;
+    font-weight: 700;
+    color: $gray-70;
+    margin-bottom: 5px;
+    text-transform: uppercase;
+  }
+
+  .value {
+    display: block;
+    width: 100%;
+    color: $black;
+  }
+
+  .separator {
+    width: 2px;
+    height: 50px;
+    background-color: $gray-20;
+  }
+}
+
+.button {
+  height: 36px;
+}
+
+@media screen and (max-width: 576px) {
+  .card-container {
+    padding: 0 1.5em;
```

Comment:
Why is it necessary to redefine the padding? @01001110J

Reply:
Responsive purposes

Reply:
Responsive purposes